util/include/Metrics.hpp

```diff
@@ -70,6 +70,8 @@ class Gauge {
   explicit Gauge(const uint64_t val) : val_(val) {}
 
   void Set(const uint64_t val) { val_ = val; }
+  void Inc(const uint64_t val) { val_ += val; }
```

Comment:
This should not be part of the public interface. Gauges are not counters and explicitly should *not* be incremented and decremented.

Reply:
See below